src/Transport/Grpc/ForwardingServerStreamingCall.php

```diff
@@ -32,17 +32,17 @@
 
 namespace Google\ApiCore\Transport\Grpc;
 
+use Grpc\ServerStreamingCall;
+
 /**
  * Class ForwardingServerStreamingCall wraps a \Grpc\ServerStreamingCall.
  *
  * @experimental
  */
 class ForwardingServerStreamingCall extends ForwardingCall
 {
-    /**
-     * @var \Grpc\ServerStreamingCall
-     */
-    protected $innerCall;
+    /** @var ServerStreamingCall */
+    protected object $innerCall;
```

Comment:
We could use exact types rather than the generic `object`, here and at other places?
```suggestion
    /** @var ServerStreamingCall */
    protected ServerStreamingCall $innerCall;
```

Reply:
I see, `tests/Tests/Unit/Transport/Grpc/ForwardingServerStreamCallTest.php` is failing and it appears like a similar property in clild class is of the type `object` too. So, lgtm.